tests/resources/module/module_inheritance/views/PrestaShop/Admin/AdvancedParameters/administration.html.twig

```diff
@@ -0,0 +1,3 @@
+{% block content %}
```

Comment:
The idea here is to override every "modern" page using the module, and check that the template inheritance works as expected.